test/org/github/jamm/testedclasses/PublicClassWithPackageProtectedClassFields.java

```diff
@@ -0,0 +1,44 @@
+package org.github.jamm.testedclasses;
+
+import org.github.jamm.MemoryMeter;
+
+public class PublicClassWithPackageProtectedClassFields
+{
+    public static String publicStaticField = "publicStaticField";
+
+    private static String privateStaticField = "privateStaticField";
+
+    public String publicField;
+
+    String packageProtectedField;
+
+    protected String protectedField;
+
+    private String privateField;
+
+    private PackageProtectedClass packageProtectedClass; 
+
+    public PublicClassWithPackageProtectedClassFields(String publicField,
+                                                      String packageProtectedField,
+                                                      String protectedField,
+                                                      String privateField) {
+
+        this.publicField = publicField;
+        this.packageProtectedField = packageProtectedField;
+        this.protectedField = protectedField;
+        this.privateField = privateField;
+        this.packageProtectedClass = new PackageProtectedClass("packageProtectedClassPublicField",
+                                                               "packageProtectedClassPackageProtectedField", 
+                                                               "packageProtectedClassProtectedField",
+                                                               "packageProtectedClassPrivateField");
+    }
+
+    public long measure(MemoryMeter meter) {
+        return meter.measure(this) 
+                + meter.measureDeep(publicField)
+                + meter.measureDeep(packageProtectedField)
+                + meter.measureDeep(protectedField)
+                + meter.measureDeep(privateField) 
+                + packageProtectedClass.measure(meter);
+    }
+}
```

Comment:
This looks ok to me, just left a small comment about naming. 
CI pushed here:
https://app.circleci.com/pipelines/github/ekaterinadimitrova2/jamm/41/workflows/87ba096d-3d3f-404e-8d5e-5760ae84ef5e

Reply:
Java 11 tests seemed to detect some size changes.

Reply:
Thanks for noting, I guess you mean the failing tests? Those were failing even before dues to some issues with Contended. More information can be found in the Readme